src/parser/normalizeValue.ts

```diff
@@ -56,7 +56,12 @@ export function normalizeValue(
     } else if (fillIndex === -1 && strokeIndex > -1) {
       ouputValue = STROKE;
     }
-  } else if (attr === 'href' || attr === 'xlink:href' || attr === 'font') {
+  } else if (
+    attr === 'href' ||
+    attr === 'xlink:href' ||
+    attr === 'font' ||
+    attr === 'id'
+  ) {
```

Comment:
this function is a bit weird.
We do carry a parse and and outputValue around and in some cases we just do an early return.
Hopefully there is a better way to fix it. All the attributes that start with a number that are not specially taken care of here are going to be wrong correct?
So any custom attribute will need to be listed here and that is not really feasible

Reply:
is it better to explicitly specify which one needs to be normalized?

Reply:
somehow this function become really convoluted yesterday i tried to look at it to expedite this fix. Eventually we can just fix as you did and open a ticket for a refactor, i didn't decide it yet sorry.